modul/achievement_notification.py

```
@@ -68,7 +68,13 @@ def draw(self, screen):
         screen.blit(bg_surface, (rect_x, rect_y))
 
         header_color = (255, 215, 0, alpha)
-        header_text = "ACHIEVEMENT UNLOCKED!"
+        try:
+            from modul.i18n import gettext
+        except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)